### PR TITLE
[collector] Managing server's resources with JMX. #582

### PR DESCRIPTION
**Pull request by koo-taejin**
Managing server's resources with JMX.

### CODE REVIEW

File: collector/src/main/java/com/navercorp/pinpoint/collector/manage/PinpointCollectorManager.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            } catch (Exception e) {
+                logger.warn("Failed to register {} MBean.", pinpontMBean);
+            }
+        }
+
+        logger.info("PinpointCollectorManager initialization completed.");
+    }
+
+    @PreDestroy
+    public void finalize() {
```
Comment (jaehong-kim) [inline collector/src/main/java/com/navercorp/pinpoint/collector/manage/PinpointCollectorManager.java:58]: Avoid finalize method name.

File: collector/src/main/java/com/navercorp/pinpoint/collector/manage/PinpointCollectorManager.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+/**
+ * @author Taejin Koo
+ */
+public class PinpointCollectorManager {
+
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    private final PinpointMBeanServer pinpointMBeanServer;
+    private final List<PinpointCollectorMBean> pinpointMBeanList = new ArrayList<PinpointCollectorMBean>();
```
Comment (jaehong-kim) [inline collector/src/main/java/com/navercorp/pinpoint/collector/manage/PinpointCollectorManager.java:36]: Where is pinpointMBeanList element ?

Comment (koo-taejin) [inline collector/src/main/java/com/navercorp/pinpoint/collector/manage/PinpointCollectorManager.java:36]: Element will be added. (ex) ROS on/off, ...)

File: collector/src/main/java/com/navercorp/pinpoint/collector/manage/PinpointCollectorManager.java (hunk 1 above)
Comment (koo-taejin) [inline collector/src/main/java/com/navercorp/pinpoint/collector/manage/PinpointCollectorManager.java:58]: thanks ~ I will change finalize method name.

**Comment by koo-taejin** [comment]
@jaehong-kim  
Edit code you mentioned. and I create new pull request with rebase source code.

**Pull request closed without merging**